tests/ImageSharp.Tests/Image/ImageTests.WrapMemory.cs

```diff
@@ -570,7 +570,6 @@ public unsafe void WrapMemory_Pointer_Null(int size, int height, int width)
         }
 
         private static bool ShouldSkipBitmapTest =>
-            !TestEnvironment.Is64BitProcess || (TestHelpers.ImageSharpBuiltAgainst != "netcoreapp3.1" &&
-                                                TestHelpers.ImageSharpBuiltAgainst != "netcoreapp2.1");
+            !TestEnvironment.Is64BitProcess;
```

Comment:
This now gives a test error. It seems that those tests were now always skipped..

Reply:
The Boolean result is not different here for 64bit builds